src/modules/Auth/features/Login/LoginForm/LoginForm.tsx

```diff
@@ -44,11 +48,20 @@ export const LoginForm = () => {
     if (signIn.fulfilled.match(result)) {
       if (fromUrl) navigate(fromUrl);
       navigateToLibrary(navigate);
+      Mixpanel.track('Login Successful');
     }
 
     if (signIn.rejected.match(result)) {
       setErrorMessage(getErrorMessage(result.payload));
     }
+
+    Mixpanel.track('Login Button click');
```

Comment:
Every click should be tracked. I believe the `onSubmit` function is called only after the validation is passed.